src/main/java/com/powsybl/network/conversion/server/NetworkConversionController.java

```diff
@@ -131,8 +133,20 @@ public ResponseEntity<NetworkInfos> importCgmesCase(@RequestParam("caseUuid") UU
     @PostMapping(value = "/networks/{networkUuid}/reindex-all")
     @Operation(summary = "reindex all equipments in network")
     public ResponseEntity<Void> reindexAllEquipments(@Parameter(description = "Network UUID") @PathVariable("networkUuid") UUID networkUuid) {
-        LOGGER.debug("reindex all equipments in network");
+        LOGGER.debug("Reindex all equipments in network");
         networkConversionService.reindexAllEquipments(networkUuid);
         return ResponseEntity.ok().build();
     }
+
+    @RequestMapping(value = "/networks/{networkUuid}/indexes", method = RequestMethod.HEAD)
```

Comment:
Change by `/networks/{networkUuid}/indexed-equipments/exist`

Reply:
changed by /networks/{networkUuid}/indexed-equipments because its HEAD request then it already check existency for me